coach.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import subprocess
+
+from six import with_metaclass
+
+from singleton import Singleton
+
+
+class Coach(with_metaclass(Singleton, object)):
```

Comment:
@jnguyen75 ooc what's the point in making this a singleton when you're only calling the static method anyways. Is it just a style thing? Conveying the use to the reader?